src/components/task-details-page/TaskDetailsPage.jsx

```diff
@@ -1,21 +1,22 @@
 import React from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import { useTasks } from "../../context/tasks-context/TaskProvider";
+import Timer from "../timer/Timer";
 import "./task-details.css";
 
 export default function TaskDetailPage() {
     const { taskId } = useParams();
+    console.log(taskId)
     const { taskList, dispatchTaskList } = useTasks();
     const navigate = useNavigate();
 
     const getTaskDetails = (list, id) => {
-        return list.find((task) => task.id === id);
+        return list.find((task) => {
+            return task.id === id});
```

Comment:
you don't need 2 return the keyword arrow will return the expression.

Reply:
done